brave/https-everywhere-builder
# Update deps and rulesets to 2018.9.19

Test plan:
1. build the rulesets
2. replace client rulesets with the newly-built rulesets to test them
3. go to http://selection.europcar.it/ and it should auto-update to HTTPS

To upload to prod, run `npm run upload --prod`
